sopt/src/main/scala/dagr/sopt/README.md

```diff
@@ -82,3 +82,8 @@ or multiple values one after each other:
 - A single value option must have a value after its option name.
 - A multiple value option must have one or more values after its option name.
 
+## Special Types
+
+Arguments of type `Option` can be cleared using the special `<None>` token.
```

Comment:
I think, looking at the code, this is incorrect.  I think it only responds to `//` for both `Option` and `Collection` arguments.